input/haleu/optimization/once-through/moga_tuning/tuning.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+import sys
+import os
+from random import uniform
+sys.path.append('../../../../../scripts')
+import dakota_input as inp
+
+counter = 0
+for c_rate in [0.3, 0.584, 0.8]:
+    variable_dict = {'pop_size':50,
+                     'mutation_rate':0.08,
+                     'crossover_rate':c_rate,
+                     'counter':counter}
+    dakota_file = f"tuning_{counter}.in"
```

Comment:
Would it make sense to change these labels to something like this for each loop (I know the number tells you where you were in the whole process, but this might make identifying what was being changed for that version easier)? I don't know if that's at all useful though, might make data processing annoying.
```suggestion
    dakota_file = f"tuning_cr_{counter}.in"
```

Reply:
It makes running the input files more annoying, because then you have to loop over different file names and numbers. For this work, only 10 input files were developed so it's not too hard to keep track of what's different in each input file. In the single-objective work (already merged) more files were created, so I created a `.csv. file to keep track of the parameters in each file.

Reply:
Ok, smart work-around!